### DISCUSSION TITLE
`pane:get_current_working_dir()` always returns `nil`

### DISCUSSION THREAD

**Discussion by crispinb** [wezterm/wezterm#3551] [General]
I call this on the pane passed to a `window-focus-changed` event. Not sure if this is the best event to use as I'm just experimenting, but in any case it always returns `nil`.

The purpose was to get the working dir to supply to new split-pane keybinds that open splits in the same working dir as the original pane.

**Comment by wez** [comment]
FWIW, the default for a split operation is to use the same working directory as the original pane, so you shouldn't need to manually override this.

If that isn't working, then more information is needed about your overall setup and environment. I'd encourage you to fill out all of the sections of https://github.com/wez/wezterm/issues/new?assignees=&labels=bug&template=bug.yml and file an issue; that way we get more information and it is easier for me to follow up.

**Comment by crispinb** [comment]
Thank you. I had (perhaps lazily) just assumed the behaviour I was seeing (new splits open in home) was the default.

I'll submit an issue then and close this to keep things tidy.
